src/code.sh

```
@@ -71,82 +83,73 @@ collect_rnaseq_metrics() {
-# Catch all false
-if [[ "$run_CollectTargetedPcrMetrics" == false ]] && [[ "$run_CollectHsMetrics" == false ]] && [[ "$run_CollectMultipleMetrics" == false ]]; then
-echo "No picard functions selected!"
-fi
-
-##### CLEAN UP #####
-
-# Upload all results files and directories in $HOME/out/eggd_picard_stats/
-dx-upload-all-outputs --parallel
+    ## Sanity checks
+    # Exit if no picard functions selected
+    if [[ "$run_CollectTargetedPcrMetrics" == "false" ]] && \
```

Shellcheck (0.10.0)

[warning] run_CollectTargetedPcrMetrics is referenced but not assigned.

(SC2154)